### ISSUE TITLE
Issue loading model for transfer learning

### ISSUE THREAD

**Issue by nollikai** [calico/baskerville#55]
I am trying out the transfer learning tutorial and I have run into an issue with running hound_transfer.py. When the model is initially loaded, the final layer has an undefined last axis, which seems to cause a ValueError later on in the code.

Below are the last two layers of the model that are printed to screen:

```
 dense_20 (Dense)            (None, 6144, 4)              7684      ['tf.nn.gelu_11[1][0]']       
                                                                                                  
 switch_reverse_1 (SwitchRe  (None, 6144, None)           0         ['dense_20[1][0]',            
 verse)                                                              'stochastic_reverse_complemen
                                                                    t[1][1]']     
```

What I suspect could be happening is that when the switch_reverse_1 layer is initialized, the length of strand_pair does not equal the value of last axis of the previous layer (4), and consequently switch_reverse_1 gets initialized with an undefined last axis at this line: 

`xrs = tf.keras.backend.switch(reverse, tf.gather(xr, self.strand_pair, axis=-1), xr)`

As a result, the code only seems to run without error if the number of targets (and the length of strand_pair) is 4. Any suggestions for how to work around this?

**Comment by nollikai** [comment]
Follow up: I was able to run hound_transfer.py without errors if I edit the params.json file so that the final layer length matches the number of targets. Is this correct usage?
